src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 17px;
+}
+
+.stars--1 :nth-child(1),
+.stars--2 :nth-child(-1n + 2),
+.stars--3 :nth-child(-1n + 3),
+.stars--4 :nth-child(-1n + 4),
+.stars--5 :nth-child(-1n + 5) {
```

Comment:
The use of `:nth-child` with a negative value in the formula (e.g., `-1n + 2`) is non-standard and may not work as expected across different browsers. It's recommended to use `:nth-child(n)` or `:nth-child(-n + x)` where `x` is the number of items you want to select.